liquibase-core/src/main/java/liquibase/changelog/DatabaseChangeLog.java

```diff
@@ -127,14 +126,6 @@ public void setPhysicalFilePath(String physicalFilePath) {
         this.physicalFilePath = physicalFilePath;
     }
 
-    public String getChangeLogId() {
-        return changeLogId;
-    }
-
-    public void setChangeLogId(String changeLogId) {
-        this.changeLogId = changeLogId;
-    }
```

Comment:
I'd consider the removal of hub to be a breaking change in itself so I personally think that it makes sense to also remove this. That said I'm not _too_ deadset on insisting on breaking changes and this could be kept without any real impact. I guess I'm pro removal but I won't insist on it.

Reply:
I don't feel strongly either way. I can't come up with a scenario where someone would depend on this and not also be broken by the removal of Hub. What do you think @filipelautert ?

Reply:
This was created 3 years ago for hub, and we never used it for anything else. We don't save it to any database table, and I can't find any usage on the extensions too - so I'm fine with removing it.
That said, I think it may break some liquibase-integration-tests - but that is expected.